en/ros/mavros_offboard.md

```diff
@@ -2,19 +2,19 @@
 
 > **Caution** Offboard control is dangerous. If you are operating on a real vehicle be sure to have a way of gaining back manual control in case something goes wrong.
 
-The following tutorial will run through the basics of offboard control through mavros as applied to an Iris quadcopter simulated in Gazebo. At the end of the tutorial, you should see the same behaviour as in the video below, i.e. a slow takeoff to an altitude of 2 meters.
+The following tutorial will run through the basics of `OFFBOARD` control through MAVROS as applied to an Iris quadcopter simulated in Gazebo with SITL running. At the end of the tutorial, you should see the same behaviour as in the video below, i.e. a slow takeoff to an altitude of 2 meters.
```

Comment:
Offboard should not be in uppercase throughout the document.

Reply:
As long as we're consistent that is fine. I've got some work in the user guide to make it like this.